tests/markdown_long_table/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, ["markdown"], { proseWrap: "never" });
```

Comment:
Could you add a test with default config so that we could clearly see the difference between different config for the same input?

```suggestion
run_spec(__dirname, ["markdown"]);
run_spec(__dirname, ["markdown"], { proseWrap: "never" });
```